packages/scss/demo/components/timelines.html

```diff
@@ -40,7 +40,7 @@ <h1>Timelines</h1>
 	<ol class="timeline u-textWarning">
 		<li class="timeline-step">
 			<span class="timeline-step-title">
-				Previous step
+				<a href="#" class="timeline-step-title-action">Previous step</a>
```

Comment:
Why creating a new nested element for the `<a>` ? Can't `span.timeline-step-title` be transformed to `a.timeline-step-title` ?

Reply:
It would be necessary to specify in the css if there is a link or a button to avoid hover effects.